ramp-frontend/ramp_frontend/tests/test_ramp.py

```diff
@@ -411,6 +413,119 @@ def test_ask_for_event_mail(client_session):
+    ramp_config = generate_ramp_config(read_config(config))
+
+    # upload file in sandbox.html
+    path_submissions = os.path.join(
+                                    ramp_config['ramp_kit_dir'],
+                                    submission_dir
+                                   )
+
+    with login_scope(client, 'test_user', 'test') as client:
+        rv = client.get('http://localhost/events/iris_test_4event/sandbox')
+        assert rv.status_code == 200
+
+        # choose file and check if it was uploaded correctly
+        path_submission = os.path.join(path_submissions, file_name)
+
+        try:
+            rv = client.post('http://localhost/events/'
+                             + 'iris_test_4event/sandbox',
+                             headers={'Referer':
+                                      'http://localhost/events/'
+                                      + 'iris_test_4event/sandbox'},
+                             data={'file': (open(path_submission, 'rb'),
+                                            file_name)},
+                             follow_redirects=False)
+
+            assert rv.status_code == 302
+            assert rv.location == 'http://localhost/events/' \
+                                  'iris_test_4event/sandbox'
+
+            # code of the saved file
+            with open(path_submission, 'r') as file:
+                submitted_data = file.read()
+        except FileNotFoundError:
+            submitted_data = None
+
+        # code from the db
+        event = get_event(session, 'iris_test_4event')
+        sandbox_submission = get_submission_by_name(session,
+                                                    'iris_test_4event',
+                                                    'test_user',
+                                                    event.ramp_sandbox_name)
+        submission_code = sandbox_submission.files[-1].get_code()
+
+        # get user interactions from db and check if 'upload' was added
+        user_interactions = get_user_interactions_by_name(session, 'test_user')
+
+        if correct:
+            # check if the code of the submitted file in the 'submission_code'
+            assert submitted_data is not None
+            assert submitted_data in submission_code
+            # check if the user_interaction was added to the db
+            assert 'upload' in user_interactions['interaction'].values
+        else:
+            assert submitted_data is None
+            assert 'upload' not in user_interactions['interaction'].values
+
+
+def test_sandbox_save_file(client_session, makedrop_event):
+    client, session = client_session
+    sign_up_team(session, 'iris_test_4event', 'test_user')
+
+    example_code = 'example content'
+
+    with login_scope(client, 'test_user', 'test') as client:
+        rv = client.get('http://localhost/events/iris_test_4event/sandbox')
+        assert rv.status_code == 200
+
+        rv = client.post('http://localhost/events/'
```

Comment:
If you could make the same change that I did above (this is the black python style).